docker/adjust_config.py

```diff
@@ -0,0 +1,45 @@
+
+import sys
+from argparse import ArgumentParser
+from typing import List
+
+import toml
+
+"""
+python3 ./docker/adjust_config.py --mode=main --file=/go/elrond-config-devnet/config.toml
```

Comment:
is it ok to set it for devnet only?

Reply:
In the Docker files, it is set for both devnet and mainnet (this is just a short documentation).